src/systems/light.js

```diff
@@ -14,13 +21,33 @@ var DEFAULT_LIGHT_ATTR = 'data-aframe-default-light';
  * @param {bool} userDefinedLights - Whether user lighting is defined.
  */
 module.exports.System = registerSystem('light', {
+  schema: {
+    shadowMapType: {default: 'pcf', oneOf: ['basic', 'pcf', 'pcfsoft']},
```

Comment:
Semantically nice. So then shadows would be enabled on the renderer if any `shadow` components are present, not based on whether `light="castShadow: true"` is set?